pypit/arflux.py

```diff
@@ -87,7 +87,7 @@ def bspline_magfit(wave, flux, var, flux_std, nointerp=False, **kwargs):
     -------
     """
     from pypit import arutils
-    invvar = (var > 0.)/(var + (var <= 0.))
+    invvar = (var > 0.)/(np.max(var,0))
```

Comment:
Credit for the new line goes to @tiffanyhsyu. 

Even after this change I'm still having issues with the bspline with my red side data so I think this part of the code still needs some work. I'm still working on understanding what's going wrong but maybe @rcooke-ast suggestion might help?